rubix-bookkeeper/src/main/java/com/qubole/rubix/bookkeeper/BookKeeperServer.java

```diff
@@ -70,7 +72,14 @@ public void run()
 
   public static void startServer(Configuration conf)
   {
-    bookKeeper = new BookKeeper(conf);
+    try {
+      bookKeeper = new BookKeeper(conf);
+    }
+    catch (FileNotFoundException e) {
+      log.error("Cache directories could not be created", e);
+      return;
+    }
```

Comment:
@abhishekdas99 The exception thrown when trying to create the cache directories will now be caught when trying to initialize BookKeeper.